content/normal-modal-logic/syntax-and-semantics/relational-models.tex

```diff
@@ -33,7 +33,7 @@
 represented by means of simple diagrams, such as the one in
 \olref{fig:simple}. Worlds are represented by nodes, and world~$w'$ is
 accessible from~$w$ precisely when there is an arrow from~$w$
-to~$w'$. Moreover, we label a node (world) by~$\mTrue{p}$ when $w \in V(p)$,
+to~$w'$. Moreover, we label a node (world) by~$p$ when $w \in V(p)$,
```

Comment:
Not an error, but `\mTrue` makes a gap in the front for a phantom `\neg`, and within the text this shows as a weird gap.

Reply:
I'll fix it in a different way. Someone might decide to label things in some other way (eg, \models p, or +p, or whatever)